.evergreen/config.yml

```diff
@@ -1,48 +1,191 @@
+# Mark a failure as a system/bootstrap failure (purple box) rather then a task
+# failure by default.
+# Actual testing tasks are marked with `type: test`
+command_type: system
+
+# Protect ourselves against rogue test case, or curl gone wild, that runs forever
+exec_timeout_secs: 3600
+
+# When a task that used to pass starts to fail, go through all versions that may have been skipped
+# to detect when the task started failing
+stepback: true
+
+#######################################
+#              Functions              #
+#######################################
 functions:
-  fetch source:
-    # Executes git clone
+  "fetch source":
+    # Executes git clone and applies the submitted patch, if any.
     - command: git.get_project
       params:
         directory: src
-    # Applies the submitted patch, if any
-    - command: git.apply_patch
 
-  exec script:
-    - command: shell.exec
-      type: test
+  "create expansions":
+    - command: subprocess.exec
+      params:
+        working_dir: src
+        binary: bash
+        args:
+          - .evergreen/create-expansions.sh
+        include_expansions_in_env:
+          - is_patch
+          - version_id
+          - project
```

Comment:
I think `args` and `binary` usually go together. Whether both show up at the end or before `include_expansions_in_env`, tbh, is not very important, as long as the convention is consistent in the same repo.

I think `include_expansions_in_env` has used array already, just as in `args`, nope?

Reply:
If it is absolutely trivial (which it seems to be), we should just make it consistent with Java, which will prevent us from having to change things there.

I do not know the yaml term, but I mean, they should be on a single line, rather than on 3 lines, similar to our convention in Java source code arguments. For example, `include_expansions_in_env: ["DRIVERS_TOOLS"]`.

Alternatively, we should always use dash-prefix multi-line format, even for a single expansion. In any case, we just need to decide what we will do in both projects.

Reply:
I prefer the dash-prefix multi-line format for sure

Reply:
I moved `args` to the bottom